package.json

```diff
@@ -356,6 +357,15 @@
         },
         "dependencies": [],
         "licenseFilename": "LICENSE.md"
+      },
+      "responsifyjs": {
+        "name": "responsifyjs",
+        "js": {
+          "dist/cropper.js": "responsify.js",
+          "dist/cropper.min.js": "responsify.min.js"
```

Comment:
Almost there. You need to check the source file, probably in node_modules/responsifyjs/dist and make that the left part of this line. Obviously cropper.js is not correct.

BTW Good job 👍

Reply:
run `npm install responsifyjs --save`, this will:
- copy all the files needed in node_modules
- add the dependency in package.json.

Then check the folder `node_modules/responsifyjs/dist` and make sure the right side of the line highlighted in the first comment is correct (probably "dist/responsify.js")